src/components/menu/js/menuController.js

```diff
@@ -118,14 +118,17 @@ function MenuController($mdMenu, $attrs, $element, $scope, $mdUtil, $timeout, $r
     triggerElement = triggerElement || (ev ? ev.target : $element[0]);
     triggerElement.setAttribute('aria-expanded', 'true');
     $scope.$emit('$mdMenuOpen', $element);
+    var hasBackdrop = getAttr('useBackdrop', true);
     $mdMenu.show({
       scope: $scope,
       mdMenuCtrl: self,
       nestLevel: self.nestLevel,
       element: menuContainer,
       target: triggerElement,
       preserveElement: true,
-      parent: 'body'
+      parent: 'body',
+      hasBackdrop: hasBackdrop,
```

Comment:
Please add a test for that. 

Additionally, I can't see the actual implementation of the `hasBackdrop` option? Is this not implemented yet?